Dockerfile

```diff
@@ -1,21 +1,21 @@
 FROM amazonlinux:2.0.20200722.0
 
 ARG STACK_VERSION=2.5.1
+ARG USER=haskell
 
-ENV LANG=C.UTF-8 \
-  LC_ALL=C.UTF-8
```

Comment:
I don't think `LC_ALL` is needed. Usually it's used as a big hammer to make sure other language settings aren't messed up. Setting `LANG` should be enough.